pkg/oc/bootstrap/docker/up.go

```diff
@@ -1112,7 +1113,7 @@ func (c *ClientStartConfig) RegisterTemplateServiceBroker(out io.Writer) error {
 // Login logs into the new server and sets up a default user and project
 func (c *ClientStartConfig) Login(out io.Writer) error {
 	server := c.OpenShiftHelper().Master(c.ServerIP)
-	return openshift.Login(initialUser, initialPassword, server, c.LocalConfigDir, c.originalFactory, c.command, out)
+	return openshift.Login(initialUser, initialPassword, server, c.LocalConfigDir, c.originalFactory, c.command, out, ioutil.Discard)
```

Comment:
It is probably debatable if this should use ioutil.Discard or not.

Reply:
cc @csrwng

Reply:
The alternative would be to wire each handler to receive a second io.Writer for errs

Reply:
why not use the same writer for both? (out)

Reply:
the more info we can return to users about failures, etc. the better

Reply:
thanks, went ahead and re-used `out` for both